WordPress/src/main/res/values/dimens.xml

```diff
@@ -483,6 +483,7 @@
     <dimen name="page_item_horizontal_padding">16dp</dimen>
     <dimen name="page_parent_radio_button_offset">11dp</dimen>
     <dimen name="one_line_list_item_height">48dp</dimen>
+    <dimen name="one_line_list_item_height_small">40dp</dimen>
```

Comment:
My guess is design may be not right here.  The default line height on [Android needs to be 48dp](https://material.io/components/lists#specs) (40dp is on iOS)

Reply:
You're right! I asked @annchichi, she'll update the height with `48dp` in the design.
Revert the style changes here: f48d24f5a4f32f380ecb52f653636f379aa20f52